MinecraftClient/config/ChatBots/discordWebhook.cs

```diff
@@ -0,0 +1,369 @@
+    public void toggleOverlay() { overlay = !overlay; }
+    public void toggleFallbackSkin() { fallbackSkin = (fallbackSkin == "MHF_Steve" ? "MHF_Alex" : "MHF_Steve"); }
+
+    public int getSize() { return size; }
+    public int getScale() { return scale; }
+    public bool getOverlay() { return overlay; }
+    public string getFallbackSkin() { return fallbackSkin; }
+    public Dictionary<string, string> getModeDict() { return skinModes; }
+}
+
+class HTTP
+{
+    public static byte[] Post(string url, NameValueCollection pairs)
+    {
+        using (WebClient webClient = new WebClient())
+        {
+            return webClient.UploadValues(url, pairs);
+        }
+    }
+}
+
+class DiscordWebhook : ChatBot
+{
+    private string webhookURL;
+    private bool sendPrivate;
+    private bool sendServer;
+    private bool getUUIDFromMojang;
+    private bool togglesending;
+    private bool allowMentions;
+    private bool onlyPrivate;
+    private Dictionary<string, string> messageContains = new Dictionary<string, string>();
+    private Dictionary<string, string> messageFrom = new Dictionary<string, string>();
+    SkinAPI sAPI = new SkinAPI();
+
+    public DiscordWebhook()
+    {
+        getUUIDFromMojang = true;
+        sendServer = true;
+        togglesending = false;
+        allowMentions = false;
+        onlyPrivate = false;
+    }
+
+    public override void Initialize()
+    {
+        base.Initialize();
+        LogToConsole("Made by Daenges.\nThank you to Crafatar for providing the beautiful avatars!");
+        LogToConsole("Please set a Webhook with '/dw changeurl [URL]' and activate the Bot with '/dw pausesending'. For further information type '/dw help'.");
+        RegisterChatBotCommand("discordWebhook", "/DiscordWebhook 'size', 'scale', 'fallbackSkin', 'overlay', 'skintype'", getHelp(), commandHandler);
+        RegisterChatBotCommand("dw", "/DiscordWebhook 'size', 'scale', 'fallbackSkin', 'overlay', 'skintype'", getHelp(), commandHandler);
+    }
+
+    public override void GetText(string text)
+    {
+        if (togglesending)
+        {
+            string message = "";
+            string username = "";
+            text = allowMentions ? GetVerbatim(text) : GetVerbatim(text).Replace("@", "[at]");
+
+            if (IsChatMessage(text, ref message, ref username) && !onlyPrivate)
+            {
+                sendWebhook(username, message);
+            }
+            else if (IsPrivateMessage(text, ref message, ref username) && sendPrivate)
+            {
+                sendWebhook(username, "[Private Message]: " + message);
+            }
+            else if (sendServer)
+            {
+                sendWebhook("[Server]", text);
+            }
+        }
+    }
+
+    public void setWebhook(string newWebhook) { webhookURL = newWebhook; }
+
+    public void setSkinAPI(string newCurrentSkinType = "flatFace", string newFallbackSkin = "MHF_Steve", int newSize = 512, int newScale = 10, bool newOverlay = true)
```

Comment:
Please capitalize the first letter of each method to match `PascalCase` naming style.